codetransformer/transformers/literals.py

```diff
@@ -42,3 +44,56 @@ def visit_STORE_MAP(self, instr):
 
         yield Instruction(ops.STORE_SUBSCR)
         # TOS  = m
+
+
+def _format_constant_docstring(type_):
+    return dedent(
+        """
+        Transformer that applies a callable to each {type_} constant in the
+        transformed code object
+
+        Parameters
+        ----------
+        f : callable
+            A callable to be applied to {type_} literals.
+        """
+    ).format(type_=type_.__name__)
+
+
+class _ConstantTransformerBase(CodeTransformer):
+
+    def __init__(self, f):
```

Comment:
`CodeTransformer.__init__` accepts an argument (there is a pr to make kwonly), so this should forward kwargs to super